src/framed/std/core.cljc

```diff
@@ -130,3 +139,15 @@
        (loop []
          ~@body
          (recur)))))
+
+(defn try-to-edn
```

Comment:
I think the naming shouldn't necessarily include "try" because in some cases there's no exception handling aspect to it, and in the other case the fact that it's exception handling is an incidental implementation detail rather than just "plainly" returning nil on invalid EDN.